src/js/modules/SectionHeader.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ArrowRight from 'js/svg/ArrowRight';
 
-const SectionHeader = ({ title }) => (
-  <h2 className="coa_SectionHeader">{title}</h2>
+
+const SectionHeader = ({ title, arrow }) => (
+  <div>
+    <h2 className="coa_SectionHeader">
+      {title}
+      {arrow && (
+        <span className="coa_SectionHeader--arrow">
```

Comment:
more picky -- should coa_SectionHeader--arrow be coa_SectionHeader__arrow, since the arrow is an element within the SectionHeader block and not a modifier?